README.md

```diff
@@ -6,7 +6,7 @@
 
 ## Getting Started ##
 
-* [Install .NET MAUI](https://github.com/dotnet/maui/wiki/Getting-Started)
+* [Install .NET MAUI](https://docs.microsoft.com/dotnet/maui/get-started/first-app?pivots=windows)
```

Comment:
```suggestion
* [Install .NET MAUI](https://docs.microsoft.com/dotnet/maui/get-started/installation)
* [Build your first .NET MAUI app](https://docs.microsoft.com/dotnet/maui/get-started/first-app)
```

Reply:
Single link, as the idea is for this to be replaced with the getting started tutorial when ready. We could point this to installation and then have installation point to first app as next step?